app/controllers/application_controller.rb

```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
+  include SessionsHelper
```

Comment:
Module khác gì class em?

Reply:
Module là một tập hợp các câu lệnh, được gói lại, dùng để mix-in(nhúng) vào một hoặc nhiều class, module khác 
Class là 1 object , thể hiện đặc điểm chung của một nhóm các object bằng cách tìm ra những thuộc tính và phuơng thức chung của object